app/controllers/account/feedbacks_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Account::FeedbacksController < Account::AccountController
+  before_action :require_super_admin_only!
+
+  def index
+    @feedbacks = Feedback.by_creation_date.all
```

Comment:
fyi: AR collection spawn does not really require `#all` to be called
`Feedback.all.by_creation_date` or `Feedback.by_creation_date` would make more sense